osctiny/extensions/attributes.py

```diff
@@ -0,0 +1,73 @@
+"""
+Attributes extension
+--------------------
+
+.. versionadded:: 0.8.0
+"""
+import typing
+from urllib.parse import urljoin
+
+from lxml.objectify import ObjectifiedElement
+
+from ..utils.base import ExtensionBase
+
+
+class Attribute(ExtensionBase):
+    """
+    Access attribute namespaces and definitions
+    """
+    base_path = "/attribute/"
```

Comment:
Would you mind removing the `/` at the end here?
```suggestion
    base_path = "/attribute"
```
Reasons given "below" (i.e. in the other review comments).